tools/run_tests/run_xds_tests.py

```diff
@@ -646,7 +751,7 @@ def get_startup_script(path_to_server_binary, service_port):
 sudo apt install -y git default-jdk
 mkdir java_server
 pushd java_server
-git clone https://github.com/grpc/grpc-java.git
+git clone -b xds_server_health https://github.com/ericgribkoff/grpc-java.git
```

Comment:
Nit: `--single-branch`?

Reply:
Oops, this was just to run tests before my PR had been merged on grpc-java (which is now done).